src/Topologies/Topologies.jl

```diff
@@ -43,9 +44,9 @@ abstract type AbstractDistributedTopology <: AbstractTopology end
 coordinate_type(topology::AbstractTopology) = coordinate_type(domain(topology))
 
 """
-    domain(topology)
+    Topologies.domain(topology::AbstractTopology)
 
-Returns the domain of the `topology` from the underlying `mesh`
+Returns the [`Domains.AbstractDomain`](@ref) of the `topology` from the underlying `mesh`
```

Comment:
```suggestion
Returns the domain (a subtype of [`Domains.AbstractDomain`](@ref)) of the `topology` from the underlying `mesh`
```

I might leave this as it was for preciseness, but we can include the ref and type information like this?